README.md

```diff
@@ -451,17 +452,19 @@ In general, startups rely on investors to help fund rapid growth.
 🄳 In order to finance building or scaling their business, startups **fundraise** by selling
 shares in their business to [investors](http://www.investopedia.com/terms/i/investor.asp) in
 exchange for capital.
-Startups that aspire to grow rapidly are likely to fundraise from
-[**venture capital**](https://en.wikipedia.org/wiki/Venture_capital) investors, who are
-individuals or firms that specialize in such investments.
-After a company IPOs and is public, it can seek investment in public markets.
+Startups that aspire to grow rapidly are likely to fundraise from individuals or firms
+specializing in startup investment, called
+[**venture capital**](https://en.wikipedia.org/wiki/Venture_capital). After a company goes public
```

Comment:
This first sentence is now technically wrong usage: Venture capital is the type of financing, not the individuals or firms.

Startups that aspire to grow rapidly are likely to fundraise from individuals or firms specializing in startup investment, in a kind of financing called **venture capital**.

Reply:
"startup investment, called venture capital" is the grammar

Reply:
Fixed.